src/tests/JIT/Regression/JitBlue/Runtime_105474/Runtime_105474.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using Xunit;
+
+#nullable disable
+
+public class Runtime_105474_A
+{
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    private void Method1()
+    {
+        Vector128<ulong> vr0 = Vector128.CreateScalar(1698800584428641629UL);
+        AdvSimd.ShiftLeftLogicalSaturate(vr0, 229);
```

Comment:
I thought we have coverage in unit test for invalid immediate too. can you double check why this problem didn't appear in those tests?

Reply:
We should validate that the `GT_HWINTRINSIC` node is marked `GTF_SIDE_EFFECT` when we insert the `GT_BOUNDS_CHK` node over `op2` here (this should be happening already as part of `GenTreeMultiOp::InitializeOperands` which runs in the constructor).

Beyond that, we likely need to also ensure that hwintrinsic nodes marked as `GTF_HW_USER_CALL` are also marked as `GTF_EXCEPT` when the allowed input range is a partial range and for that to be stripped in rationalization (where we already strip `GTF_HW_USER_CALL`) if it is determined to be "in range".